armi/physics/neutronics/tests/test_crossSectionManager.py

```diff
@@ -295,6 +295,8 @@ def test_ComponentAverage1DCylinder(self):
 
         xsgm.interactBOL()
 
+        xsgm.updateNuclideTemperatures()
```

Comment:
I guess this is just a check to see that the `NotImplementedError` doesn't arise? Not sure if it's easy to add more to this test?